Server/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -536,6 +539,52 @@ int CLuaElementDefs::getElementData(lua_State* luaVM)
     return 1;
 }
 
+int CLuaElementDefs::isElementData(lua_State* luaVM)
+{
+    //  bool isElementData ( element theElement, string key [, inherit = true] )
+    CElement* pElement;
+    SString   strKey;
+    bool      bInherit;
+
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadUserData(pElement);
+    argStream.ReadString(strKey);
+    argStream.ReadBool(bInherit, true);
+
+    if (!argStream.HasErrors())
+    {
+        CLuaMain* pLuaMain = m_pLuaManager->GetVirtualMachine(luaVM);
+        if (pLuaMain)
+        {
+            if (strKey.length() > MAX_CUSTOMDATA_NAME_LENGTH)

```

Comment:
Can you explain this a bit? I don't really understand what you mean by this.

Reply:
I was thinking maybe the
```cpp
if (strKey.length() > MAX_CUSTOMDATA_NAME_LENGTH)
{
    // Warn and truncate if key is too long
    m_pScriptDebugging->LogCustom(luaVM, SString("Truncated argument @ '%s' [%s]", lua_tostring(luaVM, lua_upvalueindex(1)),
                                                    *SString("string length reduced to %d characters at argument 2", MAX_CUSTOMDATA_NAME_LENGTH)));
    strKey = strKey.Left(MAX_CUSTOMDATA_NAME_LENGTH);
}
```
part should be after the `if (argStream.HasErrors())` block instead of inside this block. But it's optional.